using-social-media-as-communication-tool/readme.md

```diff
@@ -15,78 +18,89 @@ Here you will find some key concepts about social media that will help you to su
 
 You do not want people to only interact with your messages, but to do so in a positive manner. You want people to speak positively about you with personal incentive.
 
 This can be done by saying something about you, but just sharing or even liking your messages is enough. The best way to get your readers to do this is simply by giving them a reason to. To do this, it is important to know why people interact with messages on social media.
 
-Your social media profile is a definition of who you are. It is your public resume, your dating profile and your portfolio. So with everything you share and say you should ask yourself, 'is this portfolio material? Would I endorse this and how does it look with the rest of my profile?'
+## Why do people share?
+
+Your social media profile is a definition of who you are. It is your public resume, your dating profile and your portfolio. So with everything you share and say you should ask yourself:
+
+> Is this portfolio material? Would I endorse this and how does it look with the rest of my profile?'
 
 Because you only reach the people during the time of your broadcast, it can often be good to broadcast the same message multiple times and give the receiver a reason to share it every time.
 
-This is the main reason why non-profit organizations and education are so successful on social media with consumer goods, retail and e-commerce far behind in second place. Their messages are directly relatable to the people sharing them and lend a strong connotation to the sender and the sharer.
+This is the main reason why non-profit organisations and education institutes are so successful on social media. Their messages are directly relatable to the people sharing them and lend a strong connotation to the sender and the sharer.
 
 ## Practicality
 
 Here are some insights that have been drawn from research on social media:
 
-* In general, people only briefly look at what they see on social media. Therefore, the first sentences or image they see should capture the reader in a way that is in line with what the message you're conveying.
+* In general, people only briefly look at what they see on social media. Therefore, the first sentences or image they see should capture the reader in a way that is in line with what message you're conveying.
 
-* People hardly click on links (1%-10%), even if they have instant relevance. Therefore, you should limit use of tinyurl and bitly to urls of which the names are not relevant or obvious.
+* People hardly click on links (1%-10%), even if they have instant relevance. Therefore, you should limit use of [bitly](https://bitly.com/) and [tinyurl](http://tinyurl.com/) for urls of which the names are not relevant or obvious. These are tools to shorten your links, especially useful for twitter of course. Although your audience doesn't recognise your link, it can be handy for tracking.
 
```

Comment:
"useful for twitter of course"
This is no longer true since they all get the same amount of characters right?

Also: Twitter is with a capital